src/main/java/com/alrex/parcool/extern/shouldersurfing/ShoulderSurfingManager.java

```diff
@@ -1,37 +1,48 @@
 package com.alrex.parcool.extern.shouldersurfing;
 
 import com.alrex.parcool.common.action.impl.Dodge;
+import com.alrex.parcool.common.action.impl.Dodge.DodgeDirection;
 import com.alrex.parcool.extern.ModManager;
 import com.alrex.parcool.utilities.MathUtil;
 import com.github.exopandora.shouldersurfing.api.client.IShoulderSurfingCamera;
 import com.github.exopandora.shouldersurfing.api.client.ShoulderSurfing;
 import net.minecraft.client.Minecraft;
 import net.minecraft.client.player.LocalPlayer;
+import net.minecraft.world.phys.Vec3;
 
 public class ShoulderSurfingManager extends ModManager {
+    private static Minecraft mc = Minecraft.getInstance();
+
     public ShoulderSurfingManager() {
         super("shouldersurfing");
     }
 
     public Dodge.DodgeDirection handleCustomCameraRotationForDodge(Dodge.DodgeDirection direction) {
         if (!isInstalled()) return direction;
```

Comment:
Just I'm wondering, if all dodge roll direction is `Front` when shoulder surfing is working, method : Should `handleCustomCameraRotationForDodge` return `DodgeDirection.Front` always?